### PR TITLE
Change Windows installation instructions

### PR DESCRIPTION
**Pull request by libewa**
### Motivation:

The Windows installation process errored out on my PC, complaining that `start` could not find the `vs_community.exe` file, which was however present after running `ls`. 

### Modifications:

- Change the installation code block for Windows to not use `start`, but instead execute the installer directly.
- Change the reference to the Microsoft Store from "App Store".

### Result:

The installation instructions for Windows have changed to 
a) not quit with an error
b) correctly reference the Microsoft Store

### CODE REVIEW

**Comment by alexandersandberg** [comment]
cc: @compnerd

**Comment by stevapple** [comment]
The Windows installation guide currently handles VS installation manually. We can leverage WinGet features instead:
```
winger install Microsoft.VisualStudio.2019.Community --force --custom "--add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64"
```

Breakdown:
- The `Microsoft.VisualStudio.2019.Community` installer is exactly the latest version of `https://aka.ms/vs/16/release/vs_community.exe`;
- `--force` ensures that the installer is run even when the latest VS is installed;
- `--custom` passes the supplied flags to the installer, which installs the dependencies for us.

Such approach is much less error-prone as WinGet already counts for different environments (most notably, Powershell or CMD).

**Comment by stevapple** [comment]
Also I feel that installing the community version of VS is a waste because Swift developers don't need the IDE stuff. Visual Studio C++ Build Tools (`vs_buildtools.exe` or `Microsoft.VisualStudio.2019.BuildTools`) is pretty enough.

**Comment by compnerd** [comment]
@libewa did you run from cmd or powershell/pwsh?  The expectation is that you are using the cmd shell.  For automation, the use of `start` is important.

@stevapple yes, winget is the easier solution.  Especially now that winget is a stub on Windows 11 that auto installs.  Perhaps we should swap the order and direct people to the winget instructions?

Also, you are correct that the build tools are sufficient but there are questions around licensing.  VS Community is permitted for use for many cases.  The build tools require a license of Visual Studio and thus the installation of VS.  The instructions seem complicated enough so I think that we should keep to the Visual Studio installation.

**Comment by libewa** [comment]
@compnerd I used pscore7, maybe that's the problem, but I'm going on holiday and can't try any of the solutions.
@stevapple sounds good, as we are installing deps with winget anyways. we should maybe add a note that winget comes preinstalled on win11.

**Comment by libewa** [comment]
also, maybe we should actually use cmd syntax highlighting instead of pwsh

File: download/_windows.md [hunk 1]
```diff
@@ -36,20 +36,19 @@ Invoke-RestMethod -Url 'get.scoop.sh' | Invoke-Expression
...
+   ~~~ cmd
    winget install Git.Git
    winget install Python.Python.3.10
 
    curl -sOL https://aka.ms/vs/16/release/vs_community.exe
-   start /w vs_community.exe --passive --wait --norestart --nocache ^
-     --installPath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community" ^
-     --add Microsoft.VisualStudio.Component.Windows10SDK.19041 ^
-     --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64
+   winget install Microsoft.VisualStudio.2019.Community --force --custom "--add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64"
```
Comment (compnerd) [inline download/_windows.md:44]: I wonder if we should just move this to VS2022 at this point and update the SDK to the latest.

File: getting-started/_installing.md [hunk 2]
```diff
@@ -82,32 +82,29 @@ Invoke-RestMethod -Url 'get.scoop.sh' | Invoke-Expression
...
+   ~~~ cmd
    winget install Git.Git
    winget install Python.Python.3.10
 
    curl -sOL https://aka.ms/vs/16/release/vs_community.exe
-   start /w vs_community.exe --passive --wait --norestart --nocache ^
-     --installPath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community" ^
-     --add Microsoft.VisualStudio.Component.Windows10SDK.19041 ^
-     --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64
+   winget install Microsoft.VisualStudio.2019.Community --force --custom "--add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64"
```
Comment (compnerd) [inline getting-started/_installing.md:93]: Similar

File: download/_windows.md (hunk 1 above)
Comment (libewa) [reply download/_windows.md:42]: this is a story for another pr imo

**Review by compnerd — APPROVED** [review]
LGTM

**Review by alexandersandberg — APPROVED** [review]

**Comment by alexandersandberg** [comment]
@swift-ci please test

**Comment by alexandersandberg** [comment]
@swift-ci please test

**Pull request merged**